manifests/init.pp

```diff
@@ -1263,4 +1263,19 @@
       fail("Pam is only supported on RedHat, SuSE, Debian and Solaris osfamilies. Your osfamily is identified as <${::osfamily}>.")
     }
   }
+
+  if $ensure_vas == 'present' {
```

Comment:
Exec['vasinst'] is somewhere in the vas module. You need to include that class here to ensure that the dependency is found. This would also mean updating the README. The metadata does not need to be updated, since the vas module is not a hard requirement, though the README should not that if you want that functionality, you need Ericsson/vas >= x.y.z